corehq/apps/receiverwrapper/rate_limiter.py

```diff
@@ -162,7 +170,7 @@ def report_case_usage(domain, num_cases):
     _report_current_global_case_update_thresholds()
 
 
-def _delay_and_report_rate_limit_submission(domain, max_wait, delay_rather_than_reject, datadog_metric):
+def _delay_and_report_rate_limit_submission(domain, max_wait, delay_rather_than_reject, datadog_metric, limiter):
```

Comment:
This signature change would break any call that isn't also changed, but I checked the codebase and there are just these four calls from that one function, so 👍